src/ceph-volume/ceph_volume/tests/devices/lvm/test_prepare.py

```diff
@@ -115,6 +116,49 @@ def test_safe_prepare_osd_already_created(self, m_is_ceph_device):
             expected = 'skipping {}, it is already prepared'.format('/dev/sdfoo')
             assert expected in str(error.value)
 
+    def test_setup_device_device_name_is_none(self):
+        result = lvm.prepare.Prepare.setup_device(self=None, device_type='data', device_name=None, tags={'ceph.type': 'data'}, size=0, slots=None)
```

Comment:
probably better to keep consistency with other tests:
```suggestion
        result = lvm.prepare.Prepare([]).setup_device(device_type='data', device_name=None, tags={'ceph.type': 'data'}, size=0, slots=None)
```